### ISSUE TITLE
[DevOps] Consider configuring the app through environment variables

### ISSUE THREAD

**Issue by LucianU** [code4romania/seismic-risc#67]
Rather than hard-coding things like database host, name, user, Django secret key and other things, we could take them from the environment.

This ensures that our app follows the [12-factor app](https://12factor.net/config) design principles and makes our lives easier, because it will be easier to pass different configuration depending on the deployment.

Here is how this looks: https://testdriven.io/blog/dockerizing-django-with-postgres-gunicorn-and-nginx/

Basically, the database settings would look something like this:
```python
DATABASES = {
    "default": {
        "ENGINE": os.environ.get("SQL_ENGINE", "django.db.backends.sqlite3"),
        "NAME": os.environ.get("SQL_DATABASE", os.path.join(BASE_DIR, "db.sqlite3")),
        "USER": os.environ.get("SQL_USER", "user"),
        "PASSWORD": os.environ.get("SQL_PASSWORD", "password"),
        "HOST": os.environ.get("SQL_HOST", "localhost"),
        "PORT": os.environ.get("SQL_PORT", "5432"),
    }
}
```
We could use [django-configurations](https://django-configurations.readthedocs.io/en/stable/), which makes it easier to read from the environment. Additionally, it gives us a way to separate development and production settings.

**Comment by RaduCStefanescu** [comment]
Great proposition @LucianU

**Comment by aramboi** [comment]
This is implemented as part of PR #76
